### ISSUE TITLE
BBL Liv nicht mehr verfügbar

### ISSUE THREAD

**Issue by kedvsk** [hubsif/kodi-magentasport#37]
der Menüpunkt für Live BBL - Spiele ist nicht mehr verfügbar - passend zum Beginn der Playoffs. 
Darüber hinaus hat die Telekom ihr Login-Verfahren um die zusätzliche Eingabe eines Codes erweitert (zumindest ist das über die Telekom eigene App  so gelöst). 
Wäre schön, wenn die Probleme auch für die Version 18 von Kodi baldmöglichst gelöst werden könnten.

**Comment by uhafner** [comment]
Ja, das ist leider auch bei mir so :-(

**Comment by uhafner** [comment]
Ich habe mal ein wenig versucht, den Grund herauszufinden. Leider gibt es im Repo kein Beispiel, wie ein funktionierender Aufruf aussah. 

Wenn ich https://www.magentasport.de/api/v2/page/31 aufrufe, dann erscheint ein Live Stream nur noch unter dem Knoten `"epg"` und nicht (mehr?) unter dem Knoten `"content"`. 

```
    "epg": {
      "start_position": 8,
      "target_url": "\/epg\/31",
      "program_url": "\/bbl\/programm",
      "elements": [
        {
          "date": {
            "iso_date": "2021-05-09T10:00:00Z",
            "utc_timestamp": "1620554400",
            "original": "2021-05-09 12:00:00"
          },
          "slots": [
            {
              "slot_time": {
                "iso_date": "2021-05-22T18:15:00Z",
                "utc_timestamp": "1621707300",
                "original": "2021-05-22 20:15:00"
              },
              "is_live": true,
              "events": [
                {
                  "type": "teamEvent",
                  "target_type": "event",
                  "target": "\/event\/697927",
                  "target_url": "\/event\/alba-berlin-hamburg-towers\/697927",
                  "target_playable": true,
                  "metadata": {
                    "id": 697927,
                    "active_video_id": 290905,
                    "league_id": 2,
                    "league_identifier": null,
                    "name": "ALBA BERLIN - Hamburg Towers",

```

Die Informationen zur URL sind hier auch alle da, die müssten nur anders zusammengesetzt werden. 

Ich frage mich, oben es funktionieren würde, einfach alle epg Knoten abzulaufen und dann die mit `"is_live": true,` zu selektieren und anzuzeigen?

**Comment by uhafner** [comment]
Ich habe jetzt noch mal etwas weiter gesucht und herausgefunden, dass die ID zum eigentlichen Video vorab im EPG auch fehlt (wenn ich das richtig deute). Ich habe mich dann heute so beholfen, dass ich den Stream im normalen Browser gestartet habe, und die ID kopiert habe, und dann direkt in Zeile https://github.com/hubsif/kodi-magentasport/blob/master/default.py#L226 eingetragen habe. Dadurch konnte ich immerhin das Spiel heute sehen, auch wenn das nicht komfortabel ist.

**Comment by hubsif** [comment]
Hi!
Ich werd's mir hoffentlich bald mal ansehen können, leider ist die Zeit bei mir gerade etwas knapp...

**Comment by hubsif** [comment]
Hi!
Ich habe gerade zwei pre-releases hochgeladen, jeweils für Kodi 18 und 19. Ich hatte nicht viel Zeit es ausgiebig zu testen, bitte um Rückmeldung ob es soweit funktioniert!

**Comment by kedvsk** [comment]
Live fehlt leider nach wie vor (Kodi 18).

**Comment by hubsif** [comment]
Hm, die Live-Spiele werden jetzt direkt im Hauptmenü angezeigt und nur, wenn gerade welche laufen. Könnte es vielleicht sein, dass du getestet hast, als es gerade kein live-Spiel gab?
Ansonsten bitte mal Ausschnitte aus dem Kodi-Logfile posten.

**Comment by kedvsk** [comment]
Ja, bin davon ausgegangen, dass wie bisher unter "Live" die folgenden Spiele als Liste angezeigt werden.

**Comment by uhafner** [comment]
Habe es gerade auch ausprobiert, funktioniert wieder! Vielen Dank für die schnelle Hilfe!

**Comment by kedvsk** [comment]
Dankeschön!
Sieht unter Android und auf dem PC (W10) gut aus. Auf dem Raspberry muss zur Aktualisierung der Live-Einträge Kodi neu gestartet werden. Zwei Raspberries verhalten sich hier identisch. Alle Systeme laufen mit identischer Skin.

**Comment by uhafner** [comment]
Das Issue kann geschlossen werden.
